mindgraph-cli.py

```diff
@@ -0,0 +1,36 @@
+from mindgraph.graph import read_yaml
+from yaml import load
+from yaml.scanner import ScannerError
+import argparse
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-f", "--yaml-file", dest="files",
+                        help="one or more yaml files separated by comma",
+                        required=True)
+    return parser.parse_args()
+
+
+def main():
+    args = arg_parser()
+    file_list = args.files.split(",")
```

Comment:
Could this be a whitespace, instead? I feel like commas aren't as common of a pattern for delimiting multiple command-line arguments

Reply:
@capsulecorplab Problem with whitespace is that, it can cause confusion, if the filename itself has a space. In shell, it will be de-limited by a backslash, but still there will be a space, which python won't be able to differentiate. What do you think?
Still, if you think reading from file is going to be the only functionality of this cli, we can remove -f flag and make it as a positional param.